### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by ReTass** [zhenliuZJU/Forge4D#3]
Dear authors,
Thanks for your excellent work. I am very interested in this work. The paper states 'All stages use a consistent batch size of 8 and are trained on 8 H20 GPUs for 100,000 iterations each, and the learning rate is linearly decreasing to 1 × 10−5 at the end of the training stage. ' , how long for the training of each stage？

**Comment by zhenliuZJU** [comment]
Thank you for your interest.

The training time for each stage is around: stage 1: 1.5 day; stage 2: 2.5 days; stage 3: 2.5 days; which sums up around one week.

**Comment by ReTass** [comment]
Thank you so much for your detailed reply!
